include/nana/gui/widgets/listbox.hpp

```diff
@@ -1461,7 +1461,19 @@ the nana::detail::basic_window member pointer scheme
 		/// Returns the number of columns
 		size_type column_size() const;
 
-		void column_resizable(bool resizable);
+		/// Move column to view_position
+		void move_column(size_type abs_pos, size_type view_pos);
+
+        /// Sort columns in range first_col to last_col inclusive using the values from a row
+        void reorder_columns(size_type first_col,
```

Comment:
Whitespace letters are used for indentation here, how about considering tabs?

Reply:
Hi thanks! I could agree, but for format style questions, I assume Jinhao's decisions. At least until we make a general style guide. On the other hand see what GitHub does with tab indentation in the following lines